controllers/operators/embed/kafka_instance.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: kafka-cluster
+spec:
+  kafka:
+    version: 3.1.0
+    replicas: 1
+    listeners:
+      - name: tls
+        port: 9092
+        type: internal
+        tls: true
+        authentication:
+          type: tls
+      - name: external
+        port: 9094
+        type: nodeport
+        tls: true
+        authentication:
+          type: tls
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      default.replication.factor: 1
+      min.insync.replicas: 1
+      inter.broker.protocol.version: "3.1"
+    storage:
```

Comment:
we need to go with persistent storage (ephemeral is ok for demo, not for production)
Similar to what @memodi did here: https://github.com/netobserv/documents/pull/30/files

By the way, the storage class is something we can try to get dynamically, but we also need a fallback if we can't find a suitable one, which means we need to expose that in the CR